invenio_app_ils/circulation/receivers.py

```diff
@@ -53,6 +53,12 @@ def index_after_loan_replace_item(_, old_item_pid, new_item_pid):
 
 def send_email_after_loan_change(_, initial_loan, loan, trigger):
     """Send email notification when the loan changes."""
+    cancel_reason = loan.get("cancel_reason", None)
+    if cancel_reason == "Anonymization":
+        return
     send_loan_mail(
-        action=trigger, loan=loan, message_ctx=dict(initial_loan=initial_loan)
+        action=trigger,
+        loan=loan,
+        manual_trigger=True,
```

Comment:
I think here it should be false because it was an action, e.g. checkout
```suggestion
        manual_trigger=False,
```